### ISSUE TITLE
MediaTransportControls is overriding AudioMuteButton's accessible name (and others)

### ISSUE THREAD

**Issue by vgromfeld** [microsoft/microsoft-ui-xaml#3375]
**Describe the bug**
We cannot set a custom accessbile name to `AudioMuteButton`. The value we set is overridden by the `MediaTransportControls` implementation.
This is also breaking the automatic keyboard accelerator integration in the button tooltip.
The same behavior applies to all the other MTC buttons. The `MediaTransportControls` implementation is forcing all the values to some default we may not want.

**Steps to reproduce the bug**
See the sample app for the full repro: [VolumeAccelerator.zip](https://github.com/microsoft/microsoft-ui-xaml/files/5332633/VolumeAccelerator.zip)
The most important steps are
1. Copy the default `MediaTransportControls` style within a project
2. Change the `AudioMuteButton` by adding a custom automation name and a keyboard shortcut
```xaml
<AppBarButton
    x:Name="AudioMuteButton"
    Margin="12"
    HorizontalAlignment="Center"
    VerticalAlignment="Center"
    AutomationProperties.Name="My custom automation name"
    Style="{StaticResource AppBarButtonStyle}">
    <AppBarButton.Icon>
        <SymbolIcon x:Name="AudioMuteSymbol" Symbol="Volume" />
    </AppBarButton.Icon>
    <AppBarButton.KeyboardAccelerators>
        <KeyboardAccelerator Key="Number0" Modifiers="Control" />
    </AppBarButton.KeyboardAccelerators>
</AppBarButton>
```
3. Run the application

**Expected behavior**
The button uses *My custom automation name* as its accessible name and the tooltip displays the keyboard shortcut.

**Actual behavior**
The button uses *Mute* as its accessible name and the tooltip is not displaying the keyboard shortcut.

Here is what Accessibility Insights is seeing:
![image](https://user-images.githubusercontent.com/20152272/95182941-8ff3bf80-07c5-11eb-816a-6c17ada57763.png)

**Comment by StephenLPeters** [comment]
Looking at the code, in OnApplyTemplate we set the automation name unconditionally. This is definitely a bug but @vgromfeld you may be able to work around this by setting the automation name after the template is applied (like in the OnLoaded event).  This will require winui 3 before we can fix.

@Austin-Lamb is reach the MTC owner?

**Comment by bpulliam** [comment]
Closing stale external bug. If you are still seeing this issue, please open a new issue in the Feedback Hub so that it may gather related logs/diagnostics.
